third_party/nvidia_sdk_download/fetch_cuda_toolkit.py

```diff
@@ -19,7 +19,7 @@
 import subprocess
 import sys
 
-VERSION = "12.1.1"
+VERSION = "12.2.1"
```

Comment:
This change is outside CI btw. This broke my build and took some time to figure out what happened. Maybe announce it outside CI changes too?